package exception.trycatch;

import java.util.Scanner;

public class TryCatchEx03 {
	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		
		while (true) {
			try {
				System.out.print("> ");
				int num = sc.nextInt();
				System.out.println("입력결과 : " + num);
			} catch (Exception e) {
				System.out.println("숫자로 입력하셀래요?");
			}
		}
	}
}
